core/src/main/scala/scalaz/syntax/ApplySyntax.scala

```diff
@@ -31,7 +31,11 @@ final class ApplyOps[F[_],A] private[syntax](val self: F[A])(implicit val F: App
   /** Alias for `|@|` */
   final def ⊛[B](fb: F[B]) = |@|(fb)
 
-  ////
+  /**
```

Comment:
The `////` is significant to the build ... I think `forever` need to go above it.